index.html

```diff
@@ -3,16 +3,22 @@
   <head>
     <title>"ALL about me"</title>
     <meta name="Farida" content="HTML"/>
+    <link rel="stylesheet" href = "css/index.css">
+   
   </head>
   <body>
+
+    <section id="my name" style="text-align: center;">
 <h1>Farida Hyderi</h1>
+</section>
+
     <section id="About">
 <h2>About</h2>
-<p>My name is Farida I am very entusiastic to learn new programming langusges
+<p style= "font-family: Times New Roman; ">My name is Farida I am very entusiastic to learn new programming languages
```

Comment:
In order for the font to be displayed, you only need to add it to either the CSS or the HTML. For this project, I'd stick with just having it in the HTML since that's more commonplace.